### PR TITLE
feat: updated texts

### PR DESCRIPTION
**Pull request by amars29**

### CODE REVIEW

File: example/storybook/src/overview/Introduction/index.stories.mdx [hunk 1]
```diff
@@ -9,16 +9,17 @@ import { Canvas, Meta, Story } from '@storybook/addon-docs';
...
+Dank is different from other styling libraries like Glamour, Emotion, Styled-components, and Vanilla-extract because it focuses on React Native while still being universal and high-performing on the web. Unlike other libraries, dank's styles are calculated during compile-time **using a babel-plugin**, which reduces the amount of processing needed during runtime. This means that with Server-Side Rendering (SSR) and Static Site Generation (SSG), pages created with dank can even run without JavaScript on the client, including media queries.
 
-Dank originally started as part of NativeBase, a universal component library for both React and React Native. However, we realized that not everyone needs all the components from NativeBase, so we broke it down into two separate libraries:
+Dank originally started as part of NativeBase, a universal component library for both React and React Native. However, we realized that not everyone needs all the components from NativeBase, so we broke it down into three separate libraries:
```
Comment (rayan1810) [inline example/storybook/src/overview/Introduction/index.stories.mdx:18]: @amars29 `two` not `three`

File: example/storybook/src/overview/Introduction/index.stories.mdx [hunk 2]
```diff
@@ -70,8 +71,8 @@ While NativeBase provides a comprehensive solution for building web and mobile a
 ## Non goals
 
 - Don’t solve problems beyond styling!
-- Accessibility (check gluestack-ui)
-- Design (check gluestack-ui again!)
+- Accessibility (check universa11y.dev)
```
Comment (rayan1810) [inline example/storybook/src/overview/Introduction/index.stories.mdx:74]: We don't have to mention universa11y.dev

Comment (rayan1810) [reply example/storybook/src/overview/Introduction/index.stories.mdx:74]: @amars29

File: example/storybook/src/overview/Introduction/index.stories.mdx [hunk 3]
```diff
@@ -9,16 +9,17 @@ import { Canvas, Meta, Story } from '@storybook/addon-docs';
...
 
-Dank originally started as part of NativeBase, a universal component library for both React and React Native. However, we realized that not everyone needs all the components from NativeBase, so we broke it down into two separate libraries:
+Dank originally started as part of NativeBase, a universal component library for both React and React Native. However, we realized that not everyone needs all the components from NativeBase, so we broke it down into three separate libraries:
 
-- [dank.style](https://dank.style) - The high-performing and universal CSS-in-JS library.
-- [ui.gluestack.io](https://ui.gluestack.io/) (coming soon) - A set of ready-to-use universal components that can be added to any project and styled to fit your needs. You can even customize the components as desired.
+- [dank.style](https://dank.style) - The high-performing and universal styling library.
+- [universa11y.dev](https://universa11y.dev) (coming soon) - A collection of HOCs that adds accessibility to low-level [dank.style](https://dank.style) components or any component with the same prop type.
```
Comment (rayan1810) [inline example/storybook/src/overview/Introduction/index.stories.mdx:21]: Same , no mention of universa11y.dev @amars29

File: example/storybook/src/overview/Introduction/index.stories.mdx (hunk 1 above)
Comment (gauravguha) [reply example/storybook/src/overview/Introduction/index.stories.mdx:18]: @amars29, dont change this part. Refer to the current doc and keep this part as it is!

**Pull request merged**